changes/900.misc.rst

```diff
@@ -0,0 +1 @@
+Added quotes in order to support zsh
```

Comment:
Change notes should always be understandable without additional context, so I've edited this to clarify *where* you added quotes.